### PR TITLE
Develop to Master (Task1, Task2, fixes after Task2)

### PR DESCRIPTION
**Pull request by aleksandrbelik**

### CODE REVIEW

File: .eslintrc [hunk 1]
```diff
@@ -0,0 +1,25 @@
+{
+    "extends": [
+        "airbnb-base",
```
Comment (loginov-rocks) [inline .eslintrc]: Prefer `airbnb` instead of `airbnb-base`.

File: .eslintrc [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+        "version": "detect"
+      }
+    },
+    "env": {
+      "jest": true,
+      "browser": true
+    },
+    "rules": {
+      "comma-dangle": ["error", "never"],
+      "class-methods-use-this": ["error", { "exceptMethods": ["render"] }]
```
Comment (loginov-rocks) [inline .eslintrc]: What is the reason to except render method here?

File: package.json [hunk 3]
```diff
@@ -1,15 +1,58 @@
...
+    "@babel/core": "^7.4.0",
+    "@babel/preset-env": "^7.3.4",
+    "@babel/preset-react": "^7.0.0",
+    "babel-jest": "^24.5.0",
+    "babel-loader": "^8.0.5",
+    "babel-plugin-module-resolver": "^3.2.0",
+    "babel-plugin-transform-export-extensions": "^6.22.0",
+    "css-loader": "^2.1.1",
+    "eslint": "^5.15.3",
+    "eslint-config-airbnb-base": "^13.1.0",
```
Comment (loginov-rocks) [inline package.json]: Prefer `eslint-config-airbnb` instead of `eslint-config-airbnb-base`.

File: package.json [hunk 4]
```diff
@@ -1,15 +1,58 @@
...
+    "file-loader": "^3.0.1",
+    "jest": "^24.5.0",
+    "jest-dom": "^3.1.3",
+    "react-test-renderer": "^16.8.4",
+    "react-testing-library": "^6.0.0",
+    "style-loader": "^0.23.1",
+    "webpack-cli": "^3.2.3",
+    "webpack-dev-server": "^3.2.1"
+  },
+  "dependencies": {
```
Comment (loginov-rocks) [inline package.json]: Webpack & webpack plugins are not runtime dependencies, they should be placed within the "devDependencies" object. Switch appropriate eslint rule off for webpack configurations.

File: .eslintrc (hunk 1 above)
Comment (aleksandrbelik) [reply .eslintrc]: @loginov-rocks fixed

File: .eslintrc (hunk 2 above)
Comment (aleksandrbelik) [reply .eslintrc]: @loginov-rocks fixed

File: package.json (hunk 3 above)
Comment (aleksandrbelik) [reply package.json]: @loginov-rocks fixed

File: package.json (hunk 4 above)
Comment (aleksandrbelik) [reply package.json]: @loginov-rocks removed aslant rule for webpack.
As for "webpack" and "webpack-merge" eslint can't pass test with this rules in "devDepencies". It is generate the follow errors:
2:15  error  'webpack-merge' should be listed in the project's dependencies, not devDependencies  import/no-extraneous-dependencies
  3:17  error  'webpack' should be listed in the project's dependencies, not devDependencies        import/no-extraneous-dependencies

Comment (loginov-rocks) [reply package.json]: If you want to lint development-related stuff (you should, btw), you can add the following line at the beginning:
```
/* eslint-disable import/no-extraneous-dependencies */
```
OR you can create `configs/.eslintrc` with the following content:

```
{
  "rules": {
    "import/no-extraneous-dependencies": "off"
  }
}
```

In any case development dependencies should never appear in the runtime dependencies list.

Comment (aleksandrbelik) [reply package.json]: @loginov-rocks fixed package json and added appropriate rule to eslint config

**Comment by loginov-rocks** [comment]
@aleksandrbelik you just disabled rule for the whole project, please reread my comment once again.

**Comment by aleksandrbelik** [comment]
> @aleksandrbelik you just disabled rule for the whole project, please reread my comment once again.

moved eslint rule to webpack configs, added linebreak-style disabling

**Pull request merged**